src/jwt.ts

```diff
@@ -10,19 +11,42 @@ interface JWT {
   raw: string
 }
 
+const isExpired = (jwt: JWT) => new Date(jwt.exp * 1000) < new Date()
```

Comment:
I think this should be <= and you should compare by casting `new Date()` to a number.

```
  The "exp" (expiration time) claim identifies the expiration time on
   or after which the JWT MUST NOT be accepted for processing.  The
   processing of the "exp" claim requires that the current date/time
   MUST be before the expiration date/time listed in the "exp" claim.```

Reply:
You can compare dates without casting to numbers

Reply:
I know, but if the expiry time is specified as a unix time it only specifies the precision to a second, and a date has more precision than that.

Reply:
Won't it just expire up to a second early on the FE?

Reply:
Yeah

Reply:
Is that a problem?

Reply:
I mean it's an off-by-one. It's not a huge problem, but it's technically wrong and it's easy to fix, so IMO you might as well fix it.